src/reducers/tests/breakpoints.spec.js

```diff
@@ -11,20 +11,66 @@ import {
   getBreakpointsForSource,
   initialBreakpointsState
 } from "../breakpoints";
-import * as I from "immutable";
 
 function initializeStateWith(data) {
-  const mappedData = I.Map(data);
   const state = initialBreakpointsState();
-  return state.set("breakpoints", mappedData);
+  state.breakpoints = data;
+  return state;
 }
 
 describe("Breakpoints Selectors", () => {
   it("it gets a breakpoint for an original source", () => {
     const sourceId = "server1.conn1.child1/source1/originalSource";
-    const matchingBreakpoints = { id1: { location: { sourceId } } };
+    const matchingBreakpoints = {
```

Comment:
why is this so different?

Reply:
flow complained about all the missing properties so I added them